libraries/Microsoft.Bot.Schema/IHandoffActivity.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Schema
+{
+    /// <summary>
+    /// Control of the conversation has been transferred, or a request to transfer control of the conversation
```

Comment:
I'm not entirely clear on what this activity represents. The first part is clear; the second part is vague. Is this activity also sent to request that the receiver transfer control of the conversation? Also, how are the original participants and the new participants flagged?